src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,81 @@
+package task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> task;
+    private int counter;
```

Comment:
since .size() is O(1), I think maintaining this counter variable might be confusing rather than just calling .size()  each time. However, this is very minor in this context!